### PR TITLE
Add new directory for MineBlocks

### PR DESCRIPTION
**Pull request by Bedeutunq**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

## Further comments
The server is currently under maintenance, but will be released soon.

### CODE REVIEW

File: minecraft_servers/melonblocks/manifest.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+  "server_name": "melonblocks",
+  "nice_name": "MelonBlocks",
+  "direct_ip": "melonblocks.net",
+  "server_wildcards": [
+    "%.melonblocks.net"
```
Comment (maaaathis) [inline minecraft_servers/melonblocks/manifest.json:6]: Doesnt look like a real wildcard
image

**Review by maaaathis — CHANGES_REQUESTED** [review]

File: minecraft_servers/melonblocks/manifest.json (hunk 1 above)
Comment (Bedeutunq) [reply minecraft_servers/melonblocks/manifest.json:6]: Could you check again?

Comment (Bedeutunq) [reply minecraft_servers/melonblocks/manifest.json:6]: It should be changed now.

Comment (maaaathis) [reply minecraft_servers/melonblocks/manifest.json:6]: looks good 👍

**Comment by maaaathis** [comment]
server is in maintenance
image

**Comment by Bedeutunq** [comment]
Yeah, but the server will be released soon... 

If it doesn't work now, can you please make an exception?

**Comment by Nils-afk** [comment]
No exception was made for me either. So I very much doubt it.

**Comment by DoJapHD** [comment]
Just reopen the PR once the server is released

**Pull request closed without merging**